web/dashboard_info.js

```diff
@@ -2105,6 +2112,22 @@ netdataDashboard.context = {
 
     'unbound.cache': {
         info: 'The number of items in each of the various caches.'
+    },
+
+    'boinc.tasks': {
+        info: 'The total number of tasks and the number of active tasks.  Active tasks are those which are either currently being processed, or are partialy processed but suspended.
```

Comment:
missing '
(`info: 'text`)

Reply:
Hmm, I think I've found a bug in Vim's JS syntax highlighting then...